README.md

```diff
@@ -16,8 +16,8 @@ Stable and Canary channels are officially supported.  Beta and Dev channels will
 ## Installation
 
 * Add [Gruvbox Dark](https://chrome.google.com/webstore/detail/devtools-theme-gruvbox-da/njcgdakjdifgccdgnoiphpnihcfopcmj) from the Chrome Web Store.
-* Open `chrome://flags` &#9654; Enable Developer Tools experiments and click "Relaunch Now" at the bottom.
-* Open developer tools settings &#9654; Experiments &#9654; [&#10004;] Allow custom UI themes.
+* Open Chrome devtools settings and check "Allow extensions to load custom stylesheets".
+* Reload Chrome
```

Comment:
Thanks for your contribution. I haven't updated the repo since Chrome released its own dark theme so I didn't realize the instructions were outdated. I'm adding one more step to clarify that this option is in the Experiments tab.

```suggestion
* Open Chrome Developer Tools
* Go to Settings > Experiments tab
* Check "Allow extensions to load custom stylesheets"
* Reload Chrome
```